bc_obps/registration/fixtures/mock/operation.json

```diff
@@ -3,17 +3,14 @@
     "model": "registration.operation",
     "pk": "e1300fd7-2dee-47d1-b655-2ad3fd10f052",
     "fields": {
+      "point_of_contact": 1,
```

Comment:
I got errors when I removed this, but I don't think it's supposed to be used in reg2, to investigate